EthersJs/accelerating-development-with-ethersjs.md

```diff
@@ -0,0 +1,242 @@
+# Accelerating DApp Development with Ethersjs
+
+Today's decentralized application stack often consists of a front end component, smart contracts, and a framework to interact with the blockchain. Web3 is used today in a majority of cases to facilitate the interaction with the Ethereum blockchain, however, Web3 is a large, poorly documented, and difficult to maintain. Ethersjs is an alternative library that offers all of the features of web3 in a tiny, well-tested package.
+
+Ethersjs is used today to create a seamless connection between a frontend and the Ethereum blockchain. In this example, we employ Angular 7.X and Ethersjs to create a simple wallet application and interact with a smart contract deployed on the Ethereum blockchain.
+
+### Prerequisites
+
+First, you will need to install nodejs and Angular. Both installations can be found here:
```

Comment:
```suggestion
First, you need to install node.js and Angular. You can find both installations here:
```

Correct project name, removed unnecessary future tense (readers are doing this *now*) and passive voice. These are all grammar rules that I don't expect people to fully understand, and will correct many in a final edit, but also believe in helping tech writers learn them :)